<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStatistics</class>
 <widget class="QWidget" name="FormStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QChartView" name="ChartViewPercentages">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="LabelAveragePercent">
         <property name="text">
          <string>Средний процент: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabelSummColumn">
         <property name="text">
          <string>Всего достижений:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QChartView" name="ChartsViewTimes"/>
   </item>
   <item>
    <widget class="QChartView" name="ChartsViewMonths"/>
   </item>
   <item>
    <widget class="QChartView" name="ChartsViewYears"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
